recipes/sentry-native/all/conanfile.py

```diff
@@ -46,12 +46,12 @@ class SentryNativeConan(ConanFile):
         "with_crashpad": "sentry",
         "crashpad_with_tls": "openssl",
         "with_breakpad": "sentry",
-        "wer": False
+        "wer": False,
     }
 
     @property
     def _min_cppstd(self):
-        return "17"
+        return "17" if Version(self.version) < "0.7.7" else "20"
```

Comment:
Hi @MartinDelille , I've been testing the PR on my machine without using this version requirement, and both on Mac and Linux, version 0.7.8 works for me. I've checked upstream, and it seems that the version is 17.

https://github.com/getsentry/sentry-native/blob/master/CMakeLists.txt#L33

Maybe we can just undo this change and see how it works.

As for the other change from clang 5.1 to 10.0, I can't find a reason for it. Could you provide any justification for this change?


Sorry for the delay :(